cfme/tests/distributed/test_appliance_replication.py

```diff
@@ -14,37 +17,30 @@
 ]
 
 
-def configure_replication_appliances(appliances):
-    """Configure two database-owning appliances, with unique region numbers,
-    then set up database replication between them.
+def configure_replication_appliances(remote_app, global_app):
```

Comment:
We have a code/functionality duplicity at least  of this function with `cfme/fixtures/cli.py::replicated_appliances_with_providers`.

This function  does a subset of what the `replicated_appliances_with_providers` does. I was thinking it would be great to place yours `configure_replication_appliances` to cfme/utils/appliance/console.py to make it share-able. Then the `replicated_appliances_with_providers` could be changed to use it.

But as there already are functions for setting the other kind of replication in the `console.py` -- the HA replication, we would need to name it different. Perhaps `configure_centralized_control` as this feature is called "Replication and Centralized Administration" in the documentation.

The question is should this be done in this PR?

Reply:
I'd like to address that in a separate PR. Right now I'm just modifying some pre-existing code in a single test module; it will be a bit beyond this PR's scope to consolidate this with the fixture code and make sure it works across all tests.